src/index.js

```diff
@@ -1,13 +1,51 @@
+// eslint-disable-next-line
 import _ from 'lodash';
-import './styles/style.css'
+import './styles/style.css';
 
-function component() {
-  const element = document.createElement('div');
+const list = document.getElementById('list');
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+const toDos = [
+  {
+    description: 'Wash car',
+    completed: false,
+    index: 1,
+  },
 
-  return element;
-}
+  {
+    description: 'Play fifa',
+    completed: false,
+    index: 2,
+  },
 
-document.body.appendChild(component());
\ No newline at end of file
+  {
+    description: 'Cook',
+    completed: false,
+    index: 3,
+  },
+];
+
+list.innerHTML = `
+  <article>
+    <h2>Today's todo </h2>
+    <iconify-icon icon="bx:refresh"></iconify-icon>
+  </article>
+  <hr>
+  <form>
+    <input type="text" id="new-todo" class="no-outline" placeholder="Add to your list">
+    <button type="submit" id="add-todo" value><iconify-icon icon="uil:enter"></iconify-icon></button>
+  </form>
+  <hr>
+`;
+
+toDos.forEach((toDo) => {
+  const listItem = document.createElement('li');
+  listItem.classList.add('task');
+  listItem.innerHTML = ` 
+    <input type="checkbox">
+    <label>${toDo.description}</label>
+    <iconify-icon icon="ph:dots-three-outline-vertical-fill" class="dots"></iconify-icon>
+    <hr>  
+  `;
+
+  list.append(listItem);
+});
```

Comment:
- [ ] Kindly add the code that iterates over the tasks inside a function as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Write%20a%20function%20to%20iterate%20over%20the%20tasks%20array%20and%20populate%20an%20HTML%20list%20item%20element%20for%20each%20task.).